docker/conda/environments/cuda11.8_dev.yml

```diff
@@ -31,11 +31,12 @@ dependencies:
     - click >=8
     - cmake=3.24
     - configargparse=1.5
+    - cuda-compiler=11.8
     - cuda-nvml-dev=11.8
-    - cuda-python<=11.7.0 # Remove when Issue #251 is closed
     - cudatoolkit=11.8
-    - cudf 22.10
+    - cudf 23.02
     - cupy=9.5.0
+    - cxx-compiler
```

Comment:
With the addition of this package, it automatically brings in `gxx_linux-64` and `gcc_linux-64`.
```
$ mamba repoquery depends cxx-compiler

Executing the query cxx-compiler



 Name         Version Build       Channel    
──────────────────────────────────────────────
 cxx-compiler 1.5.2   hf52228f_0  conda-forge
 c-compiler   1.5.2   h0b41bf4_0  conda-forge
 gxx          11.2.0  h702ea55_10 conda-forge
 gxx_linux-64 11.2.0  hacbe6df_10 conda-forge
```
Should we remove them from our dependencies?